libs/product/state/src/actions/product.actions.ts

```diff
@@ -7,11 +7,23 @@ import { DaffProduct } from '@daffodil/product';
  * Action types for Product Actions.
  */
 export enum DaffProductActionTypes {
+    ProductLoadByUrlAction = '[Product] Load By URL Action',
     ProductLoadAction = '[Product] Load Action',
     ProductLoadSuccessAction = '[Product] Load Success Action',
     ProductLoadFailureAction = '[Product] Load Failure Action',
 }
 
+/**
+ * Triggers a request for a Product.
+ *
+ * @param payload - URL of requested Product
+ */
+export class DaffProductLoadByUrl<T extends DaffProduct = DaffProduct> implements Action {
```

Comment:
Can you think of an example when this would be used? I can't

Reply:
Honestly I'm not even sure what the purpose of the non-page load actions are so I'll defer to you on this.

Reply:
removed

Reply:
It would be a similar usage as loading a category for a home page or something. Probably less useful since I would think that loading categories that aren't on category pages is a lot more common than loading a single product on a page that isn't for that particular product.